code/modules/modular_computers/file_system/programs/arcade.dm

```diff
@@ -101,7 +101,7 @@
 	if(computer)
 		printer = computer.all_components[MC_PRINT]
 
-	var/gamerSkillLevel = user.mind?.get_skill_level(/datum/skill/gaming)
+	var/gamerSkillLevel = usr.mind?.get_skill_level(/datum/skill/gaming)
```

Comment:
also remove the mob/user from the proc arguments, `/datum/proc/ui_act(action, list/params, datum/tgui/ui, datum/ui_state/state)`

Reply:
Done!